packages/input/src/input.vue

```diff
@@ -299,12 +302,15 @@
         this.focused = true;
         this.$emit('focus', event);
       },
-      handleCompositionStart() {
-        this.isComposing = true;
-      },
-      handleCompositionEnd(event) {
-        this.isComposing = false;
-        this.handleInput(event);
+      handleComposition(event) {
```

Comment:
don't combine into one handler. write separate ones for composition{start,update,end}.

Is compositionUpdate handler really necessary?

Reply:
Fixed it and pushed new commit. 

> Is compositionUpdate handler really necessary?
As i know, yes.
When compositionstart event trigger, `event.target.value` is empty. So we dont know whether input language is Korean or not.